lua/harpoon/term.lua

```diff
@@ -72,4 +72,9 @@ M.sendCommand = function(idx, cmd, ...)
     end
 end
 
+M.clear_all = function()
+    log.trace("clear_all(): Clearing all terminals.")
```

Comment:
I think we should go through each terminal and call `vim.api.nvim_buf_delete`.  That should at least remove it from the buffers menu too!

Reply:
Got it. My initial thought was to mimick the behavior of `clear_all()` for marks and leave the buffer pool untouched but I'll make that change.